### ISSUE TITLE
Duplicate "Read more" links in spotlight section on health services page

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#12081]
## Describe the defect
On VAMC System health services pages, there is a spotlight section.  This features 3 items and each has a "Read more" link. These links however are not unique so a non-visual user may not fully understand the purpose of them.

## AC / Expected behavior

- [ ] The "Read more" links should be programmatically unique
- [ ] Accessibility review 

## Screenshots
![image](https://user-images.githubusercontent.com/106678594/209158193-5f3329cd-89f3-44a1-a887-c50e25e44d41.png)

### Possible solution(s)

1. Screen reader only text containing the title of the featured item can be added after "Read more"
   1. This will have a class of "sr-only" so it will not be shown visually but will be available to assistive technology
   2. ![image](https://user-images.githubusercontent.com/106678594/209159287-7be98ef1-8004-4418-b97f-b1ea461b4454.png)
2. Add an `aria-describedby` attribute to the link to point to the title of the featured item
  1. This indicates to assistive technology that the link and the title are associated and will be read as such
  2. ![image](https://user-images.githubusercontent.com/106678594/209160219-d9b6b63d-144f-4bf5-9e6c-729f8f5f7bb4.png)

### CMS Team
Please check the team(s) that will do this work.

- [ ] `Program`
- [ ] `Platform CMS Team`
- [ ] `Sitewide Crew`
- [ ] `⭐️ Sitewide CMS`
- [ ] `⭐️ Public Websites`
- [x] `⭐️ Facilities`
- [ ] `⭐️ User support`

**Comment by davidmpickett** [comment]
Removing the Lovell label as this is true on all VAMC pages

**Comment by laflannery** [comment]
I looked into this and met with @mmiddaugh to determine next steps. Created multiple new tickets for those:

- Standard pattern would be implemented on VBA, notes in this ticket (need to see if VBA content model ticket already exists)
- [Investigate VAMC System discrepancies](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/13801)
- [Update existing patterns to preferred standard pattern](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/13804) 
- [Front end update to underline links](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/13808)

**Comment by kmariepat-cityfriends** [comment]
Closing this card:
This card will get addressed with this work: Standardize spotlight patterns in CMS facility content types #13804

**Comment by CherylEvans** [comment]
This ticket was recreated in the va.gov-team repo Ticket #[64847](https://app.zenhub.com/workspaces/collaboration-cycle-reviews-61d89e9e5f4c120011a0ad37/issues/gh/department-of-veterans-affairs/va.gov-team/64847)
